index.js

```diff
@@ -0,0 +1,108 @@
+window.addEventListener("load", () => {
+  const numButtons = document.querySelectorAll(".wButton");
+  const opButtons = document.querySelectorAll(".rButton");
+  const eqButton = document.querySelector(".eButton");
+  const numDisplay = document.querySelector(".numDisplay");
+  const opArray = ["add", "subtract", "multiply", "divide"];
+  let currentString = numDisplay.textContent;
+
+  let currentNum = Number(currentString);
+  let firstNum = "";
+  let currentOp = "";
+  let memNum = "";
+
+  for (let n = 0; n <= 9; n++) {
+    numButtons[n].addEventListener("click", () => {
+      if (currentString.length <= 10) {
+        numDisplay.textContent = currentString + numButtons[n].textContent;
+        currentString = numDisplay.textContent;
+        currentNum = Number(currentString);
+      } else {
+        return;
+      }
+    });
+  }
+
+  numButtons[10].addEventListener("click", () => {
```

Comment:
You put unique ids on all the buttons, so why not get the decimal button by its id instead of taking the 11th button in the number buttons array?

Reply:
Because that would require another variable declaration? But you're right that the IDs are completely unused and therefore superfluous. Will remove from the next PR.